hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -269,7 +269,7 @@ private void deleteFile() {
             tempFile = null;
         }
 
-        private void readEachLine(final String line, final double innerCounter, final double totalFileSize) {
+        private void readEachLine(final String line, final double innerCounter, final double totalNumberOfLines) {
```

Comment:
`totalNumberOfLines` does not have to be a `double`, right? maybe can change the type of the variable then it does not have to be auto-boxed to `double`.